### ISSUE TITLE
Automatic updating players [permission group from Luckperms]

### ISSUE THREAD

**Issue by sakulmore** [plan-player-analytics/Plan#2269]
### Describe the issue

Hello, if i change a group to a player, and the player hasn't connect anymore, the group (in playerbase -> player list) isn't changed.

So, example:
I set a group Admin to player Player1. The player after connect to the server is updated his group in the Plan.

Now i change his group to Default (i using LuckPerms), and if player hans't connect to the server anymore, the group in Plan hasn't change. Is it changed only if he is connect to the server (and has changed group)

**Comment by AuroraLS3** [comment]
I guess this needs some listening for luck perms events

**Comment by sakulmore** [comment]
> I guess this needs some listening for luck perms events

So, can be fixed please?

**Comment by AuroraLS3** [comment]
Yeah I'll have to check if it has those events first.

**Comment by AuroraLS3** [comment]
The data is not retroactively corrected - try triggering the change in luckperms again for the data to update

**Comment by sakulmore** [comment]
I tried it several times, but still not working.

**Comment by AuroraLS3** [comment]
Yeah but since my fix attempt didn't fix it I'll have to try something else.

Since the ticket does **not** have the "Won't fix" label, you can safely assume that it will be fixed at some point

**Comment by sakulmore** [comment]
If i can ask, when will be the plugin updated?

**Comment by AuroraLS3** [comment]
I don't know.
I have a full time job and a thesis to write. I do this as a hobby.
